modules/automatically-scaling-machines-to-available-bare-metal-hosts.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/managing-bare-metal-hosts.adoc
+
+[id="automatically-scaling-machines-to-available-bare-metal-hosts_{context}"]
+= Automatically scaling machines to the number of available bare metal hosts
+
+You can configure `Machine` objects to be automatically created to match the number of available `BareMetalHost` objects. This is done by adding a `metal3.io/autoscale-to-hosts` annotation to the `MachineSet` object that you want to be scaled automatically.
```

Comment:
Passive voice. Maybe: To create the number `Machine` objects that match the number of available `BareMetalHost` objects automatically, add a `metal3.io/autoscale-to-hosts` annotation to the `MachineSet` object.